tool.ts

```diff
@@ -229,12 +228,35 @@ export async function downloadTool(url: string, fileName?: string): Promise<stri
                 throw err;
             }
 
+            const contentLength: number = Number(response.message.headers['content-length']);
+            if(contentLength) {
+                tl.debug(`content-length: ${contentLength} bytes`);
+            } else {
+                tl.debug(`content-length header missing.`);
+            }
+
             tl.debug('creating stream');
             let file: NodeJS.WritableStream = fs.createWriteStream(destPath);
             file.on('open', async (fd) => {
                 try {
                     let stream = response.message.pipe(file);
                     stream.on('finish', () => {
+                        let fileSize: number;
+                        try {
+                            fileSize = fs.statSync(destPath).size;
+                            tl.debug(`file size: ${fileSize} bytes`);
+                        } catch (err) {
+                            tl.debug(`Unable to find file size for ${destPath}`);
```

Comment:
If you know a reason we might not be able to stat a file we successfully wrote to (virus scanner?), it would be good to add a comment.